app/assets/javascripts/models/tour.js

```diff
@@ -0,0 +1,38 @@
+var Tether = require('tether');
+var Shepherd = require('tether-shepherd');
+
+module.exports = Backbone.Model.extend({
+  name: 'tour',
+  initialize: function(){
+  },
+  create: function() {
+    return new Shepherd.Tour({
```

Comment:
What is the rationale behind having a `create` method in a instance? I think all those methods from `Shepherd.Tour` could be encapsulated on the `Tour` instance. Suggestions:

- Move the creation of the `Shepher.Tour` instance to the `Tour` model constructor (like `this.shepherd = new Shepher.Tour...`;
- Add `start`, `next` and `complete` instance methods to the `Tour` model that delegates to `this.shepherd`.